packages/components/src/select-control/README.md

```diff
@@ -127,6 +127,12 @@ If this property is added, a label will be generated using label property as the
 - Type: `String`
 - Required: No
 
+### hideLabelFromVision
```

Comment:
Minor: there's an extra level of nesting in this Readme so this heading should be an `h4` instead of `h3`.